browser/src/react-redux/App/Nav.react.js

```diff
@@ -1,22 +1,24 @@
-import React from 'react'
-import { Link } from 'react-router-dom'
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Logout from './Logout.React';
 
-const User = () => {
-  return (
-    <nav>
-      <ul>
-        <li>
-          <Link to='/'>Home</Link>
-        </li>
-        <li>
-          <Link to='/signup'>Sign Up</Link>
-        </li>
-        <li>
-          <Link to='/login'>Login</Link>
-        </li>
-      </ul>
-    </nav>
-  )
-}
+const User = () => (
+  <nav>
+    <ul>
+      <li>
+        <Link to="/">Home</Link>
+      </li>
+      <li>
+        <Link to="/signup">Sign Up</Link>
+      </li>
+      <li>
+        <Link to="/login">Login</Link>
+      </li>
+      <li>
+        <Logout />
```

Comment:
Did we talk about conditionally rendering these based on whether the user was logged in or not? I forgot where we left off with that conversation.

Reply:
Yeah, that is the plan.  We can talk about how to go about that next week and create a new issue.

Reply:
Done.